Feat  add unsubscibe combination

blackholeearth opened the pull request:
- #36

blackholeearth committed:
add UnsubscribeCombination

KeyCombo , add Test

This reverts commit 3e26e6407387013f9741a25bfaa3c1792239bdc3.

added UnsubscribeCombination

added UnsubscribeCombination

add doc to  UnsubscribeCombination.

blackholeearth committed:
add ignore files - TestPolyFilll/bin - obj

TestPolyFilll projj - ignore files fix.

Empiree commented:
Hi @blackholeearth !

Sorry for the late reply! I have a very busy schedule at the moment, but I will try to consider your changes and bring them into the project!

Thanks so much for the contribution!

blackholeearth committed:
restore .gitignore

Revert "Delete .gitignore"

This reverts commit 977279bbff3adc20e1a7689360a8d26018f5b60f.

Revert "add ignore files - TestPolyFilll/bin - obj"

This reverts commit c50a7831cd1b0082c221dfe675f8c52ab96cf898.
